client/scripts/uploader.js

```diff
@@ -1,7 +1,17 @@
+var is_correct_dimensions = function(url) {
```

Comment:
I see that the function below is snake_cases as well, but camelCase would be better IMO.

Also is it possible to use `const` instead of `var`? I don't see why we would want to reallocate this variable later.
But maybe it is a too old version of js being used here.

Reply:
No, it is not possible. This is an old version of javascript that doesn't support const (const was added in es6 in 2015: https://www.w3schools.com/js/js_const.asp). Template strings was added in the same version, and Meta-Tv won't build when using them: https://github.com/datasektionen/Meta-TV/commit/ff051f6dc27bd6c0f1acb1083925f3a6626fe099